NitroxModel/NitroxModel.csproj

```diff
@@ -47,9 +47,16 @@
     <Reference Include="LZ4">
       <HintPath>..\lib\LZ4.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>F:\SteamLibrary\steamapps\common\Subnautica\Subnautica_Data\Managed\Newtonsoft.Json.dll</HintPath>
```

Comment:
Fix path

Reply:
If my mind serves me correctly, you can just use `Include="Newtonsoft.Json"` and get rid of the `SpecifcVersion` element as well.